vm/vm_test.go

```diff
@@ -536,10 +545,8 @@ func runInterruptTest(t *testing.T, script string) {
 	}()
 
 	_, err = env.Execute(script)
-	if err == nil {
-		t.Errorf("Execute error - received %v - expected: %v", err, ErrInterrupt)
-	} else if err.Error() != ErrInterrupt.Error() {
-		t.Errorf("Execute error - received %v - expected: %v", err, ErrInterrupt)
+	if err == nil || err.Error() != ErrInterrupt.Error() {
```

Comment:
Let's remove this part at this PR.

Reply:
May I ask why? All that is doing it removing an extra if statement.

Reply:
It makes it difficult for me to find issue that problem related to a bug.

Reply:
@mattn 

Would be happy to understand why this change makes it difficult to find the issue. If you could please let me know why, that would be great.

Here is what I could see:

The job https://travis-ci.org/mattn/anko/jobs/356702744 said:
--- FAIL: TestInterrupts (0.11s)
	vm_test.go:540: Execute error - received <nil> - expected: Execution interrupted

Can look at line 540 here:
https://github.com/mattn/anko/blob/72fa4855a49189e82452215a4cc4cb22b16b2b8c/vm/vm_test.go#L540

That line is in the function runInterruptTest, which will runs these tests:
https://github.com/mattn/anko/blob/72fa4855a49189e82452215a4cc4cb22b16b2b8c/vm/vm_test.go#L445-L507

So the change in the if statements will not make it any easier to figure out which test was the problem, right?